components/merkledb/src/proof_map_index/proof.rs

```diff
@@ -270,9 +279,9 @@ pub struct MapProof<K, V> {
 /// See [`MapProof`] for an example of usage.
 ///
 /// [`MapProof`]: struct.MapProof.html#workflow
-#[derive(Debug, Serialize, Deserialize)]
-pub struct CheckedMapProof<K, V> {
-    entries: Vec<(K, Option<V>)>,
+#[derive(Debug, Serialize)]
```

Comment:
I'm not sure why `Serialize` / `Deserialize` traits are needed for checked proofs. Since `Deserialize` is not possible with borrowed data in the struct, I've removed it.